pkg/config/config.go

```diff
@@ -207,6 +207,9 @@ type EngineConfig struct {
 	// memory.
 	EnablePortReservation bool `toml:"enable_port_reservation,omitempty"`
 
+	// Environment variables to be used when running the container engine. For example "http_proxy=internal.proxy.company.com"
```

Comment:
Are these intended to be passed to Podman and Buildah? We need to make that more clear, not everyone uses our terminology about container engine vs runtime.

Reply:
Good point! Could you open an issue for that? I guess we need to run over the docs/configs/etc. and look out for that.

Reply:
They will be passed to Podman, Buildah. Are you suggesting something like this ?
```suggestion
	// Environment variables to be used when running the container engine (e.g., Podman, Buildah). For example "http_proxy=internal.proxy.company.com"
```

Reply:
Yeah, that would be good!